CONTRIBUTING.md

```diff
@@ -64,3 +80,19 @@ Updating the package requires a fork of the repository and a push request [examp
 ### Adding yourself as a rsconnect-jupyter conda-forge maintainer
 
 Add your github username under recipe-maintainers in the [meta.yaml](https://github.com/conda-forge/rsconnect-jupyter-feedstock/blob/master/recipe/meta.yaml) file.
+
+# Deployment
```

Comment:
'Deployment' is probably not the right word. This section of rsconnect-python on 'Versioning and Releasing' is worth bringing over verbatim:

https://github.com/rstudio/rsconnect-python/blob/master/CONTRIBUTING.md#versioning-and-releasing

Reply:
I noticed we don't use a CHANGELOG in this project. Should I add one?

Reply:
Probably no for now, since changes have been in low volume and there will be new publishing solution.